src/components/form-elements/_text-input.scss

```diff
@@ -1,3 +1,4 @@
+// sass-lint:disable no-vendor-prefixes
```

Comment:
can you disable this rule only for single lines that need it, not the whole file?

Reply:
sure